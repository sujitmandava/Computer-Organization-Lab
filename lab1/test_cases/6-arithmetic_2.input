	.data
a:
	1
d:
	7
n:
	3
