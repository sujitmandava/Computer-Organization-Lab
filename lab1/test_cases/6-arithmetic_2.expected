Main Memory Contents:

65533		: 15
65534		: 8
65535		: 1
